db_schema/migrations/12_anchor_impressions.sql

```diff
@@ -0,0 +1,38 @@
+-- Create table for storing aggregate impression metrics
+CREATE TABLE IF NOT EXISTS anchorImpressions (
+    account_id INTEGER NOT NULL,
+    date_start TIMESTAMP NOT NULL,
+    date_end TIMESTAMP NOT NULL,
+    total_impressions INTEGER NOT NULL,
+    total_considerations INTEGER NOT NULL,
+    total_streams INTEGER NOT NULL,
+    considerations_conversion_rate DECIMAL(10,5),
```

Comment:
as this is percent and is always smaller than 0 I would suggest to store the value as a promille, so just unsigned 4 digits are fine?

Reply:
2 vs. 6 bytes, but ok, DECIMAL(9,5) would be at least just 5 bytes, or (5,3) would be also sufficient for a percent